File: CS/XtraSpellCheckerGetStarted/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace XtraSpellCheckerGetStarted
{
    public partial class Form1 : Form
    {
        string sText = "Hans had servd his master forseven years, so he said to to him, \"Master, my time is up; now I should be glad to go back home to my mother; give me me my wages.";
        public Form1()
        {
            InitializeComponent();
            textBox1.Text = sText;
            memoEdit1.Text = sText;
            textEdit1.Text = sText;
            richTextBox1.Text = sText;
            richEditControl1.Text = sText;
            
        }
        #region #simpleButton1_Click
        private void simpleButton1_Click(object sender, EventArgs e)
        {
            spellChecker1.CheckContainer(this);
        }
        #endregion #simpleButton1_Click
        #region #Form1_Load
		private void Form1_Load(object sender, EventArgs e)
        {
            // Set spell checking options.
            spellChecker1.SpellCheckMode = DevExpress.XtraSpellChecker.SpellCheckMode.AsYouType;
            spellChecker1.ParentContainer = this;
            spellChecker1.CheckAsYouTypeOptions.CheckControlsInParentContainer = true;
            // Set focus to the RichTextBox control to show red wavy lines under misspelled words.
            richTextBox1.Select();
        }
		#endregion #Form1_Load
    }
}

File: VB/XtraSpellCheckerGetStarted/Form1.vb
Imports System
Imports System.ComponentModel
Imports System.Drawing
Imports System.Windows.Forms

Namespace XtraSpellCheckerGetStarted

    Public Partial Class Form1
        Inherits Form

        Private sText As String = "Hans had servd his master forseven years, so he said to to him, ""Master, my time is up; now I should be glad to go back home to my mother; give me me my wages."

        Public Sub New()
            InitializeComponent()
            textBox1.Text = sText
            memoEdit1.Text = sText
            textEdit1.Text = sText
            richTextBox1.Text = sText
            richEditControl1.Text = sText
        End Sub

#Region "#simpleButton1_Click"
        Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs)
            spellChecker1.CheckContainer(Me)
        End Sub

#End Region  ' #simpleButton1_Click
#Region "#Form1_Load"
        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs)
            ' Set spell checking options.
            spellChecker1.SpellCheckMode = DevExpress.XtraSpellChecker.SpellCheckMode.AsYouType
            spellChecker1.ParentContainer = Me
            spellChecker1.CheckAsYouTypeOptions.CheckControlsInParentContainer = True
            ' Set focus to the RichTextBox control to show red wavy lines under misspelled words.
            richTextBox1.Select()
        End Sub
#End Region  ' #Form1_Load
    End Class
End Namespace
